### PR TITLE
[iOS] Linking of installed lexical models with embedded KMW

### PR DESCRIPTION
**Pull request by jahorton**
This PR passes off any relevant installed lexical models to KMW whenever a matching keyboard is activated.  I have tested this through the Simulator along with an adhoc model.kmp file and can confirm successful loading of its model within the LMLayer when these changes are applied onto #1707.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/150120:id/kvk/index.html)

**Comment by darcywong00** [comment]
I see the base branch `ios-load-model` has been updated with master. A similar merge here will make the PR easier to review.

File: ios/engine/KMEI/KeymanEngine/Classes/Manager.swift [hunk 1]
```diff
@@ -307,29 +315,20 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
...
-    
-    log.info("Setting language: \(lm.fullID)")
+  /// Registers a lexical model with KMW.
+  public func registerLexicalModel(_ lm: InstallableLexicalModel) -> Bool {
+    log.info("Setting lexical model: \(lm.fullID)")
     
     currentLexicalModelID = lm.fullID
     
-    inputViewController.setLexicalModel(lm)
+    inputViewController.registerLexicalModel(lm)
```
Comment (mcdurdin) [inline ios/engine/KMEI/KeymanEngine/Classes/Manager.swift]: This looks strange. Why are you doing this?

Comment (jahorton) [reply ios/engine/KMEI/KeymanEngine/Classes/Manager.swift]: kmwembedded.ts:

```
  /**
   * Register a lexical model
   * 
   * @param {com.keyman.text.prediction.ModelSpec} model  Spec of the lexical model
   */
  keymanweb['registerModel']=function(model: com.keyman.text.prediction.ModelSpec) {
    keymanweb.modelManager.register(model);
  };
```

It more directly parallels the nomenclature in KMW and better reflects how lexical models are managed therein.

**Pull request merged**